sql/moz-fx-data-shared-prod/telemetry_derived/attributed_windows_desktop_funnel_installs_v1/query.sql

```diff
@@ -0,0 +1,66 @@
+CREATE TEMP FUNCTION bucket_country(country STRING) AS (
+  CASE
+  WHEN
+    country IN ('US', 'CA', 'DE', 'FR', 'GB')
+  THEN
+    [country, 'tier-1']
+  WHEN
+    country IN ('MX', 'BR')
+  THEN
+    [country, 'non-tier-1']
+  ELSE
+    ['non-tier-1']
+  END
+);
```

Comment:
Should we create a UDF to do this?

Reply:
I don't think we should create a user-facing mozfun UDF out of this, since it involves logic that's pretty specific to this dataset. We could factor it out to a UDF under shared-prod, though, in which case I think it should have a name that makes it clear that it's for this use case (perhaps `desktop_funnel_bucket_country`).

Reply:
tier-1 and non-tier-1 are common operations, but what's weird is this returns an array. I could see having a public function `get_country_tier` which returns a string, and then this specific function uses it to return `[country, get_country_tier(country)]`

Reply:
I think I'm going to skip this tier-level stuff for the initial dataset. It's not clear to me that it's needed in the Looker world.